webui/react/src/services/utils.ts

```diff
@@ -20,6 +20,8 @@ export const isAuthFailure = (e: any): boolean => {
 /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
 export const isLoginFailure = (e: any): boolean => {
   return e.response && e.response.status && e.response.status === 403;
+  // TODO: Use the following when upgrading to new API login.
```

Comment:
could we capture both in one function to help us with the transition? 

isAuthFailure might need the same treatment. 

update: I just saw we have a point on this in the sync notes